src/main/java/raythx98/duke/task/TaskList.java

```diff
@@ -0,0 +1,71 @@
+package raythx98.duke.task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import raythx98.duke.exception.InvalidIndexException;
+
+/**
+ * Deals with manipulation on the list of tasks
```

Comment:
Should this JavaDoc comment end with a full stop?

Reply:
Agreed, will resolve